src/ResourceManager/Profile/Commands.Profile/Models/RMProfileClient.cs

```diff
@@ -280,6 +283,12 @@ private AzureEnvironment MergeEnvironmentProperties(AzureEnvironment environment
             SecureString password,
             ShowDialog promptBehavior)
         {
+            if (account.Type == AzureAccount.AccountType.AccessToken)
+            {
+                tenantId = tenantId ?? "Common";
```

Comment:
nit: can be moved to `SimpleAccessToken` and have an 2nd constructor that takes account and defaults tenant to `"Common"`